tutoraspects/templates/aspects/apps/scripts/aspects/clickhouse-demo-xapi-data.sh

```diff
@@ -4,8 +4,6 @@ python3 -m venv virtualenv
 . virtualenv/bin/activate
 
 echo "Loading demo xAPI data..."
-pip install git+https://github.com/openedx/xapi-db-load@0.2#egg=xapi-db-load==0.2
-pip install pandas  # clickhouse_connect is missing this
```

Comment:
If you want to review this it seems to be working now without pandas: https://github.com/openedx/xapi-db-load/pull/32 we can upgrade this to 0.3 after this PR.